### PR TITLE
feature: Enable Artwork + Order NOVO Routes

### PR DESCRIPTION
**Pull request by icirellik**
This change converts the `artwork` and `orders` routes to the novo
template.

[Review App](https://novo.artsy.net)

### CODE REVIEW

File: src/v2/Apps/Artwork/ArtworkApp.tsx [hunk 1]
```diff
@@ -46,6 +46,30 @@ export interface Props {
 
 declare const window: any
 
+export class LegacyArtworkDllContainer extends React.Component {
```
Comment (damassi) [inline src/v2/Apps/Artwork/ArtworkApp.tsx:49]: Can we add a comment above here describing whats going on?

Comment (icirellik) [reply src/v2/Apps/Artwork/ArtworkApp.tsx:49]: 🧙 and ✨ , yes I will attempt to write a comment that provides value.

Comment (damassi) [reply src/v2/Apps/Artwork/ArtworkApp.tsx:49]: 👍 ty! If we can add a bit more detail about approach (using dll, etc) in the PR description (when out of WIP) that would be appreciated too.

**Comment by damassi** [comment]
Noticing a console error on [this page](https://novo.artsy.net/artwork/balint-zsako-untitled-75): 

Screen Shot 2021-02-19 at 1 54 45 PM

**Comment by icirellik** [comment]
@damassi it might be related to caching because the DLL isn't hashed, clearing the cache fixed it for me. I'll try adding a hash.

**Comment by icirellik** [comment]
Looks like I need to add ReCaptcha too!

**Comment by damassi** [comment]
Just tried disabling cache from network inspector and doing the "empty cache and hard reload" trick but no go 🤔

**Comment by icirellik** [comment]
@damassi I don't have any issues on my end, and we know that our CDN gives people in varying locations different results. Let's wait until it's hashed to jump to conclusions!

File: src/v2/Apps/Order/orderRoutes.tsx [hunk 2]
```diff
@@ -25,6 +25,7 @@ const OrderApp = loadable(() => import("./OrderApp"), {
 // * `render` functions requires casting
 export const orderRoutes: RouteConfig[] = [
   {
+    // TODO: Still need order2?
```
Comment (joeyAghion) [inline src/v2/Apps/Order/orderRoutes.tsx:28]: 👀

File: src/v2/Apps/getAppRoutes.tsx [hunk 3]
```diff
@@ -83,7 +83,6 @@ export function getAppRoutes(): RouteConfig[] {
     // {
     //   routes: viewingRoomRoutes,
     // },
-
```
Comment (joeyAghion) [inline src/v2/Apps/getAppRoutes.tsx:86]: This is just an empty array now, right? Looking forward to the cleanup! 🍾

File: yarn.lock [hunk 4]
```diff
@@ -105,7 +105,7 @@
     ip "1.1.5"
     mailcheck "1.1.1"
     passport "0.3.2"
-    passport-apple "git+https://github.com/artsy/passport-apple.git#f41adb7822c8344b72bc36a7d68312f6592cb14f"
+    passport-apple "https://github.com/artsy/passport-apple#f41adb7822c8344b72bc36a7d68312f6592cb14f"
```
Comment (joeyAghion) [inline yarn.lock:108]: Unrelated?

**Review by joeyAghion — APPROVED** [review]

File: yarn.lock (hunk 4 above)
Comment (icirellik) [reply yarn.lock:108]: Yarn keeps changing this.

**Comment by icirellik** [comment]
This is ready for testing again and the review app has been updated. I've brought over as little CSS as I could without manually verifying each style.

File: webpack/envs/clientCommonConfig.js [hunk 5]
```diff
@@ -136,6 +136,7 @@ export const clientCommonConfig = {
   },
   optimization: {
     // Extract webpack runtime code into it's own file
```
Comment (damassi) [inline webpack/envs/clientCommonConfig.js:138]: I think we'll need to move this comment down

**Comment by damassi** [comment]
Inquiry and view in room looks good when going directly to an artwork for the first time and clicking inquire or view in room. 

inquiry

However, it looks like the JS is reiniting multiple times if navigating away (within the SPA) and then back. 

Repo: 
- https://novo.artsy.net/collect
- filter by `inquiry` artworks
- click an artwork
- click inquire
- all good
- click `artworks` top nav item, leading back to https://novo.artsy.net/collect
- filter by `inquiry` artworks
- click an artwork
- click inquire
- one will see two instances of the form appear, showing the background darker / doubling up
- this repeats depending on how many times one navigates to artwork page in SPA
- if exiting SPA and then reentering artwork page, will reset behavior
- the same applies to view in room and other legacy features
- This is due to reinitialization in artwork `componentDidMount`: https://github.com/artsy/force/pull/7064/files#diff-af5cb0bf1a4343208f4ec8e8d3cf72d23e5629f0bd449d97df4592ca0a4a0dc3R63-R67
- I fixed this by only initializing once, when `ArtworkApp` chunk loads and executes when `/artwork/id` route matches: https://github.com/artsy/force/pull/6933/files#diff-af5cb0bf1a4343208f4ec8e8d3cf72d23e5629f0bd449d97df4592ca0a4a0dc3R38-R44

Other issues: 
- Biddable artwork. https://novo.artsy.net/artwork/horst-ademeit-surrealism. Missing CSS and error: 

Screen Shot 2021-02-28 at 5 00 20 PM

Related to above, we'll want to confirm that [each of these legacy mediator js bits](https://github.com/artsy/force/blob/16591a6541228617f28b32afe317fad892e22628/src/desktop/apps/artsy-v2/apps/artwork/artworkClient.tsx#L59-L83) fire correctly and that the CSS is applied:
- `openBuyNowAskSpecialistModal`
- `openAuctionAskSpecialistModal`
- `openCollectorFAQModal`
- `openAuctionFAQModal`

**Comment by icirellik** [comment]
The flows that I am aware of appear to be working in the Review app. Will update once more when I fix the failing test.

**Comment by damassi** [comment]
Just retested and things look good! 🚀
